src/commands.jl

```diff
@@ -37,10 +37,17 @@ function show_breakpoint(io::IO, bp::BreakpointRef)
     println(io)
 end
 
+const always_run_recursive_interpret = Ref(false)
+no_chance_of_breaking() = isempty(JuliaInterpreter._breakpoints) && !JuliaInterpreter.break_on_error[]
```

Comment:
Any reason not to use `JuliaInterpreter.breakpoints()` here?
Might also make sense to check if the breakpoints are actually enabled:
```
function no_chance_of_breaking()
  bps = JuliaInterpreter.breakpoints()
  !JuliaInterpreter.break_on_error[] && (isempty(bps) || all(bp -> !bp[].isactive, bps))
end
```

Reply:
> Any reason not to use JuliaInterpreter.breakpoints() here?

It does a copy, which I don't really need. But of course, that copy doesnt matter and might as well use the official API.

Reply:
>  Might also make sense to check if the breakpoints are actually enabled:

Good idea.

Reply:
:+1: 
Unrelated, but that's not the third or so function that lives in Juno _and_ Debugger. Maybe we should think about moving those upstream (or into DebuggerUtils.jl :P).

Reply:
FWIW, I think some code duplication is fine.